include/matrix_operations/EigenvaluesCorrelation.h

```diff
@@ -0,0 +1,176 @@
+#ifndef VOLESTI_EIGENVALUES_CORRELATION_H
+#define VOLESTI_EIGENVALUES_CORRELATION_H
```

Comment:
What is the need of this new header file? Sorry, it is not clear tome.
Why the current files do not suffice to do the computations you need?

Reply:
This header file is used to try between different libraries without modifying EigenvaluesProblems.h.
Membership testing is changed to Eigen::LDLT and linear_positive_intersection is changed to Eigen::GeneralizedSelfAdjointEigenSolver